package.json

```diff
@@ -283,6 +286,7 @@
     "redux-saga-test-plan": "^4.0.6",
     "remote-redux-devtools": "^0.5.12",
     "shelljs": "^0.8.5",
+    "strip-ansi": "^6.0.0",
```

Comment:
should this go in a renovate ignore since we can't use v7 yet? I guess we could also let renovate open a PR for v7 and then close it and a new one won't be opened until there's a v8.

Reply:
Yes letting renovate try it is fine probably.